packages/bull-vault/src/interface/IBullStrategy.sol

```diff
@@ -2,6 +2,8 @@
 
 pragma solidity =0.7.6;
 
+import { ILeverageBull } from "./ILeverageBull.sol";
```

Comment:
I like this, we can add `is ILeverageBull` and not copy paste functions interface from now on